Support/Python/tbdata/brush_lookup.py

```diff
@@ -11,51 +11,67 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+# mypy: no-strict-optional
 
 import os
 import re
+from typing import cast, Dict, Iterator, List, Tuple, NewType
 from collections import defaultdict
 
+Guid = NewType('Guid', str)
+
 
 class BrushLookup():
   """Helper for doing name <-> guid conversions for brushes."""
+
+  def iter_standard_brush_guids(self) -> Iterator[Guid]:
+    """Yields all the standard (non-experimental) brush guids"""
+    with open(os.path.join(self.dir, 'Assets/Manifest.asset')) as inf:
+      data = inf.read().replace('\r', '')
+    brush_chunk = re.search(r'Brushes:\n(  -.*\n)*', data).group(0)
+    for match in re.finditer(r'guid: ([0-9a-f]{32})', brush_chunk):
+      yield cast(Guid, match.group(1))
+
   @staticmethod
-  def iter_brush_guid_and_name(tilt_brush_dir):
+  def iter_brush_guid_and_name(tilt_brush_dir: str) -> Iterator[Tuple[Guid, str]]:
+    """Yields (guid, name, is_experimental) tuples."""
     for brush_dir in ("Assets/Resources/Brushes", "Assets/Resources/X/Brushes"):
       for r, _, fs in os.walk(os.path.join(tilt_brush_dir, brush_dir)):
         for f in fs:
           if f.lower().endswith('.asset'):
             fullf = os.path.join(r, f)
-            with open(fullf) as f:
-              data = f.read()
-            guid = re.search('m_storage: (.*)$', data, re.M).group(1)
+            with open(fullf) as inf:
+              data = inf.read()
+            guid = cast(Guid, re.search('m_storage: (.*)$', data, re.M).group(1))
             # name = re.search('m_Name: (.*)$', data, re.M).group(1)
+            name = f[:-6]
```

Comment:
that conflation of `f` above and `f` here was probably my fault, from #35. Thanks!